source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md

```diff
@@ -0,0 +1,310 @@
+---
+title: Up and Running with oVirt 3.6
+author: jbrooks
+date: 2016-02-29 14:00:00 UTC
+tags: ovirt, gluster, centos
+published: false
```

Comment:
You can merge a post, and if the published flag is set to published: false, it won't be built/deployed. I'll change this.